bin/run.sh

```diff
@@ -22,5 +22,7 @@ cp -R $input_folder/* .
 
 find . -mindepth 1 -type f | grep 'Test.java' | xargs -I file sed -i "s/@Ignore(.*)//g;s/@Ignore//g;" file
 
-java -jar /opt/test-runner/autotest-runner.jar $problem_slug
+echo -e "test {\n  maxHeapSize = '2G'\n  reports.html.required = false\n}" >> build.gradle
```

Comment:
Another important change in this PR, disabling unused reporting as mentioned here:
https://docs.gradle.org/current/userguide/performance.html#report_generation

and increasing the heap for the tests makes them much faster.

Reply:
👍